<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arqiver::PrefDialog</class>
 <widget class="QDialog" name="prefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QCheckBox" name="winSizeBox">
     <property name="text">
      <string>Remember window &amp;size on closing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="startSizeLabel">
     <property name="text">
      <string>Start with this size:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinX">
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string notr="true">x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinY">
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="iconSizeLabel">
     <property name="text">
      <string>Size of view icons:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="iconSizeCombo">
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QCheckBox" name="promptBox">
     <property name="text">
      <string>Show prompt on removing items</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="tarLabel">
     <property name="toolTip">
      <string>Leave empty for the default.

Warning: Arqiver will not work if the
binary does not belong to libarchive.</string>
     </property>
     <property name="text">
      <string>libarchive binary:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="tarLineEdit">
     <property name="toolTip">
      <string>Leave empty for the default.

Warning: Arqiver will not work if the
binary does not belong to libarchive.</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QToolButton" name="tarButton">
     <property name="text">
      <string notr="true">...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QLabel" name="promptLabel"/>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>prefDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
